<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblAdmin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID Admin</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Admin</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jenis Kelamin</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblIdAdmin">
      <property name="text">
       <string>ID Admin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editIdAdmin"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblNama">
      <property name="text">
       <string>Nama Admin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editNama"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblJK">
      <property name="text">
       <string>Jenis Kelamin</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cmbJK">
      <item>
       <property name="text">
        <string>Laki - Laki</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Perempuan</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblUser">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editUser"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblPass">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editPass"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblStatus">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cmbStatus">
      <item>
       <property name="text">
        <string>PNS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pelajar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mahasiswa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wiraswasta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Guru / Dosen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TNI / POLRI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dokter / Perawat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ibu Rumah Tangga</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lainnya</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>491</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CARI DATA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
